src/domain/timeline/calendar/CalendarDialog.tsx

```diff
@@ -25,7 +25,7 @@ export interface CalendarDialogProps {
   open: boolean;
   journeyId: string | undefined;
   onClose: () => void;
-  parentPath?: string;
+  parentPath: string;
```

Comment:
It looks like we don't need that prop (passing an empty string in all instances).